test/lesson8.js

```diff
@@ -0,0 +1,14 @@
+const findMax= require('../solutions/lesson8.js');
+const test = (obj,result)=>{
+  if(findMax(obj)==result){
+    console.log(`correct, the result is ${findMax(obj)}`);
+  }else{
+    console.log(`wrong the result is ${result}`);
+  }
+}
+const a = {val:64,next:{val:75,next:{val:90,next:null}}};
```

Comment:
Add a test case for all negative numbers.

Reply:
all done, thankss! :)